rocAL/rocAL_pybind/amd/rali/pipeline.py

```diff
@@ -240,22 +241,36 @@ def copyImage(self, array):
         b.raliCopyToOutput(
             self._handle, np.ascontiguousarray(out, dtype=array.dtype))
 
+    def copyToHipTensorNHWC(self, array, multiplier, offset, reverse_channels, tensor_dtype):
+        if tensor_dtype == types.FLOAT:
+            b.raliCopyToHipOutputTensor32(self._handle, ctypes.c_void_p(array.data_ptr()), types.NHWC,
```

Comment:
make changes as per the review comments for c++ api

Reply:
Done